CMake/CopyVcpkgDlls.cmake

```diff
@@ -0,0 +1,19 @@
+# Add an extra step to copy built DLLs on MSVC
```

Comment:
We should do this automatically in Vcpkg!

Could you let me know what specifically isn't working?

Reply:
Nevermind, I see below that you're not using the toolchain file.